docs/topics/mta-rn-known-issues-7-0-0.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-0_{context}"]
+= Known issues
+
+MTA version 7.0.0 has the following issues.
+
+.Cannot create a Jira instance behind a proxy
+
+Cannot create a Red Hat Jira instance (issues.stage.redhat.com) through a proxy. link:https://issues.redhat.com/browse/MTA-849[MTA-849]
```

Comment:
No. Why are you still mentioning Red Hat Jira instance? We went through this last week